.github/workflows/ci-parity.yml

```
@@ -0,0 +1,35 @@
+name: Run parity tests
+
+on:
+  push:
+    branches: [main, "release/*"]
+  pull_request:
+    branches: [main, "release/*"]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  pytester:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Install LitServe
+        run: |
+          pip --version
+          pip install .  -U -q
+          pip install -r _requirements/test.txt -U -q
+          pip install torchvision jsonargparse -q
```

```suggestion
          pip install . "torchvision" "jsonargparse" -U -q -r _requirements/test.txt -U -q
```
resolves all compatibilities